spring-cloud-context/src/main/java/org/springframework/cloud/context/restart/RestartEndpoint.java

```diff
@@ -114,7 +114,7 @@ public ResumeEndpoint getResumeEndpoint() {
 		return new ResumeEndpoint();
 	}
 
-	@ConfigurationProperties("endpoints")
+	@ConfigurationProperties("endpoints.pause")
```

Comment:
This line was added to make sure `endpoints.enabled` still worked. Is that still the case?